CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+project(variant CXX)
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Any particular reason this isn't the first line?

Reply:
nope, fixed :)